server/forms.py

```diff
@@ -243,6 +243,17 @@ class EnrollmentForm(BaseForm):
     role = SelectField('Role', default=STUDENT_ROLE,
                        choices=ROLE_DISPLAY_NAMES.items())
 
+class SectionAssignmentForm(BaseForm):
+    name = StringField('Name', validators=[validators.optional()])
+    email = EmailField('Email',
+                       validators=[validators.required(), validators.email()])
+    sid = StringField('SID', validators=[validators.optional()])
```

Comment:
This should be validated as an `int` since you are restricting the route parameter as an integer to `/course/<int:cid>/section/<int:sctn_id>`